libraries/AP_Logger/AP_Logger_Block.h

```diff
@@ -50,26 +75,49 @@ class AP_Logger_Block : public AP_Logger_Backend {
     struct PACKED PageHeader {
         uint32_t FilePage;
         uint16_t FileNumber;
+#if BLOCK_LOG_VALIDATE
+        uint32_t crc;
+#endif
+    };
+
+    struct PACKED FileHeader {
+        uint32_t utc_secs;
```

Comment:
Sooo... the RTC may always return false if you don't have a time source - you do realise that?  This still works on a Copter with no GPS etc?

Reply:
Yes, its no worse than the status quo that has no time information at all